api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -791,4 +791,38 @@ public void getCountOfDrugs_shouldReturnTheTotalNumberOfMatchingNumbers() {
 		assertEquals(new Integer(1),
 		    conceptService.getCountOfDrugs(phrase, conceptService.getConcept(conceptId), true, true, true));
 	}
+
+	/**
+	 * @see ConceptServiceImpl#saveConceptProposal(ConceptProposal) 
+	 */
+	@Test
+	public void saveConceptProposal_shouldReturnSavedConceptProposalObject() {
+		ConceptProposal conceptProposal = new ConceptProposal();
+		conceptProposal.setOriginalText("OriginalText");
+		List<ConceptProposal> existingConceptProposals = conceptService.getConceptProposals("OriginalText");
+		assertTrue(existingConceptProposals == null || existingConceptProposals.isEmpty());
```

Comment:
Can you assert one condition instead of one, on the above line?

Reply:
@dkayiwa Those two conditions are mutually exclusive, if `existingConceptProposals` == `null`, it'll throw a NPE when we test for `existingConceptProposals.isEmpty()`. Likewise, when `existingConceptProposals.isEmpty()` is true, `existingConceptProposals` can't be null.

Reply:
You code means that we are not sure of whether getConceptProposals will return an empty list or null.

Reply:
That's true. After researching what getConceptProposals returns, ultimately `SessionImpl#list()` would be called, always returning a list object as it is initialized by `Collections#EMPTY_LIST`, therefore I am retaining `existingConceptProposals.isEmpty()` and removing the other condition. Thanks for the comments!